lms/services/moodle.py

```diff
@@ -20,6 +20,10 @@ def __init__(self, lms_url: str, token: str, http: HTTPService) -> None:
         self._token = token
         self._http = http
 
+    @property
+    def token(self):  # pragma: no cover
```

Comment:
Just changing `self._token` to  `self.token` doesn't play nice with the testing mocking.